skimage/exposure/exposure.py

```diff
@@ -505,3 +505,49 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     ratio = (limits[1] - limits[0]) / (dlimits[1] - dlimits[0])
 
     return ratio < fraction_threshold
+
+
+def hist_map(old_hist, new_hist):
+    """
+    Build a function that maps from one distribution onto another.
```

Comment:
Please expand the docstring. As it is now, it is not completely clear how to use the function.